### ISSUE TITLE
Use translation service/tool

### ISSUE THREAD

**Issue by fluks** [bpierre/gtranslate#76]
Now that the addon is localized, how about using a translation service? Github has three such [integrations](https://github.com/integrations/feature/localization). I haven't used any of those, only [Weblate](https://weblate.org/) which is used by Osmand - the Android map app - developers.

Pros:
* Non-technical people can easily provide translations.

Cons:
* Possibly lots of work for little benefit. There is not so much to translate in this project and the vocabulary is easy, so maybe programmers care to translate.

I volunteer to research about the possibilities and also to integrate the service. Mostly out of curiosity.

**Comment by bpierre** [comment]
I agree that it’s possibly a lot of work for little benefit, but it doesn’t hurt at all, so if you are interested, go for it! 👍
